.travis.yml

```diff
@@ -99,7 +100,7 @@ branches:
   - master
 
 before_install:
-- docker run --rm --privileged vicamo/binfmt-qemu:bionic
+- docker run --rm --privileged vicamo/binfmt-qemu:xenial
```

Comment:
Should move forward to disco as possible, not backward to xenial.

Reply:
Wait https://cloud.docker.com/repository/docker/vicamo/binfmt-qemu to finish building images.